docs/_config.yml

```diff
@@ -27,6 +27,7 @@ sidebar_categories:
     - features/creating-directives
     - features/authentication
     - features/testing
+    - features/plugins
```

Comment:
I feel like we should rearchitect the sidebar -- features seems like it's getting too long. What do you think?